api/src/paths/project/{projectId}/survey/{surveyId}/observation/submission/upload.test.ts

```diff
@@ -128,7 +128,7 @@ describe('uploadSubmission', () => {
   it('should throw a 400 error when no sql statement returned', async () => {
     sinon.stub(db, 'getDBConnection').returns(dbConnectionObj);
 
-    sinon.stub(survey_occurrence_queries, 'insertSurveyOccurrenceSubmissionSQL').returns(null);
+    sinon.stub(survey_occurrence_queries, 'insertSurveyOccurrenceSubmissionSQL').returns(SQL`some query`);
```

Comment:
I don't think we want to change this test.

It was covering the case when the sql returned null, which is no longer covered now.

Reply:
oh oops ...